styles.css

```diff
@@ -339,7 +343,8 @@ a:focus-visible {
     .settings-input__field--checkbox {
       width: 30px;
       height: 30px;
-
+    }
+    .settings-fields__widget .settings-input__field--checkbox {
```

Comment:
What's the point here? It increases the specificity of the selector. Is it need it?